src/app/util/ember-compatibility-functions.cpp

```diff
@@ -885,6 +874,17 @@ CHIP_ERROR WriteSingleClusterData(const SubjectDescriptor & aSubjectDescriptor,
         return apWriteHandler->AddStatus(attributePathParams, Protocols::InteractionModel::Status::InvalidValue);
     }
 
```

Comment:
This part is checking the data before calling attribute access override so that no need to do in every access and in fact inside the access, we can not send any error yet.

There is a CI test at src/controller/python/test/test_scripts/cluster_objects.py which send a too long location and except to get an error. Without these changes, we can not pass it